src/components/multi_select_state_utils.js

```diff
@@ -37,6 +37,50 @@ export const getMinMaxIndexes = (currentIndex, firstItemShiftSelected) =>
     ? { minIndex: currentIndex, maxIndex: firstItemShiftSelected }
     : { minIndex: firstItemShiftSelected, maxIndex: currentIndex };
 
+export const getAvailableIntervalForSelection = (
+  interval,
+  outsideSelectedItems,
+  maxSelectedItems,
+  firstItemShiftSelected
+) => {
+  const { minIndex, maxIndex } = interval;
+  const shouldBeSelect = outsideSelectedItems + (maxIndex - minIndex);
+  return maxSelectedItems > 0 && maxSelectedItems <= shouldBeSelect
```

Comment:
why you need to check > 0 ? is it not enough to do it like this:
maxSelectedItems && maxSelectedItems <= shouldBeSelect

Reply:
Updated